Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'gds-api-adapters', '40.1.0'
 gem 'airbrake', '4.0.0'
 gem 'logstasher', '0.6.2'
 
-gem 'govuk_navigation_helpers', '2.4.0'
+gem 'govuk_navigation_helpers', '~> 3.0.0'
```

Comment:
The commit message mentions version `3.0.1`, which is the latest. Should it be here?